index.js

```diff
@@ -13,6 +14,11 @@ function addHeaders(req) {
 }
 
 function processRefreshResponse(err, res) {
+	// Prior to OAuth 2 support the refreshcookie route returns 404.
+	if(res.status == 404) {
+		global.D2LOAuth2Disabled = true;
+		return;
+	}
 
 	// In the future we should log an error
 	if(err || !res.ok) {
```

Comment:
Yeah, interesting. Worth exploring further once we have a use case to handle it.